tests/sentry/api/endpoints/relocations/test_index.py

```diff
@@ -130,141 +128,163 @@ def test_good_status_in_progress(self):
 
     def test_good_status_pause(self):
         self.login_as(user=self.superuser, superuser=True)
-        response = self.client.get(f"{self.path}?status={Relocation.Status.PAUSE.name}")
+        response = response = self.get_success_response(
+            status=Relocation.Status.PAUSE.name, status_code=200
+        )
 
-        assert response.status_code == status.HTTP_200_OK
         assert len(response.data) == 1
         assert response.data[0]["status"] == Relocation.Status.PAUSE.name
 
     def test_good_status_success(self):
         self.login_as(user=self.superuser, superuser=True)
-        response = self.client.get(f"{self.path}?status={Relocation.Status.SUCCESS.name}")
+        response = response = self.get_success_response(
```

Comment:
Fix typo:

```suggestion
        response = self.get_success_response(
```

Reply:
whoops, I mass replaced to do the refactor so must've replaced one too many times